CHANGES.md

```diff
@@ -11,6 +11,10 @@ after the corresponding tracker issues.
 
 ## Changes since the last release
 
+- infrastructure: Upgrade GitHub Actions to Windows Server 2019 
+  (Visual Studio Enterprise 2019) and Windows Server 2022 (Visual Studio 
+  Enterprise 2022).
```

Comment:
I would add "Remove Windows Server 2016 because GitHub no longer supports it."

Reply:
In our requirements (https://www.fast-downward.org/ForDevelopers/Requirements?highlight=%28Windows%29)
we state that we support the latest two windows versions from GitHub Actions. Thus, just stating that we upgrade (to follow our guidelines for versions) would be sufficient for me. But if you want, I can explicitly state that 2016 became deprecated.

Reply:
"ForDevelopers" is means as internal documentation. It's good to be explicit about this. Also, didn't it became *unavailable*? "Deprecated" just means that you're encouraged to no longer used it because it's expected to go away in the future.